json-schemas/interface-methods/permissions-grant.json

```diff
@@ -63,12 +63,28 @@
           "type": "string"
         },
         "scope": {
-          "$ref": "https://identity.foundation/dwn/json-schemas/permissions/defs.json#/definitions/scope"
+          "oneOf": [
+            {
+              "$ref": "https://identity.foundation/dwn/json-schemas/permissions/scopes.json#/definitions/protocols-configure-scope"
```

Comment:
Sanity: I think this is no long needed on our earlier discussion? If you agree, please remove.